### ISSUE TITLE
[Errno 2]No such file or directory:"./property"

### ISSUE THREAD

**Issue by ahong007007** [deepinsight/insightface#164]
使用自己的数据集fine-tuning模型，mxnet使用官网提供的im2rec.py只能生成.lst和.rec,.idx文件，无property文件。
但是执行README.md给出的命令：
CUDA_VISIBLE_DEVICES='0,1,2,3' python -u train_softmax.py --network m1 --loss-type 12 --lr 0.005 --mom 0.0 --per-batch-size 150 --data-dir ../datasets/faces_ms1m_112x112 --pretrained ../model-m1-softmax,50 --prefix ../model-m1-triplet

提示[Errno 2]No such file or directory:"./property"，不知道这个property文件如何生成，还请回复，多谢。

**Comment by nttstar** [comment]
<nrof_classes>,112,112

**Comment by heshao90** [comment]
what's the meaning of nrof_classes?

**Comment by MengWangTHU** [comment]
@heshao90 number of classes 种类数

**Comment by KumailHussain** [comment]
@heshao90 how you solved the error , i am getting same error

**Comment by Ehtasha** [comment]
@KumailHussain  Have you resolve this ? I'm also facing same issue
